src/Compute/Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1

```diff
@@ -2650,3 +2650,35 @@ function Test-VirtualMachineScaleSetAssignedHost
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Test the VMSS Extension rolling upgrade cmdlet.
+This is a LiveOnly test and requires some manual setup due to test resources deleting themselves
+before the extension upgrade can complete. 
```

Comment:
This test requires some manual setup because the test framework does not create healthy VMs when creating a VMSS. The VMs start deleting themselves immediately, and the upgrade will fail.